app/src/main/java/com/hiddenramblings/tagmo/AmiiboDictionary.java

```diff
@@ -761,7 +764,7 @@
         charList.put(0x00130000, "Daisy");
         charList.put(0x00140000, "Waluigi");
         charList.put(0x00170000, "Boo");
-        charList.put(0x01000000, "Link (Ocarina of Time)");

```

Comment:
This character identifier seems to be used for a number of different "flavors" of Link now, not just Ocarina of Time.